File: Python/Logic-2/close_far.py
def close_far(a, b, c):
  if (abs(a - b) <= 1):
    return abs(a - c) >= 2 and abs(b - c) >= 2
  if (abs(a - c) <= 1):
    return abs(a - b) >= 2 and abs(b - c) >= 2
  if (abs(b - c) <= 1):
    return abs(a - c) >= 2 and abs(b - a) >= 2  
  return False

File: Java/Logic-2/closeFar.java
public boolean closeFar(int a, int b, int c) {
  if (Math.abs(a - b) <= 1) {
    return Math.abs(a - c) > 1 && Math.abs(b - c) > 1;
  }
  if (Math.abs(b - c) <= 1) {
    return Math.abs(a - c) > 1 && Math.abs(c - b) > 1;
  }
  if (Math.abs(a - c) <= 1) {
    return Math.abs(a - b) > 1 && Math.abs(c - b) > 1;
  }
  return false;
}
